cli/src/lib.rs

```diff
@@ -376,30 +372,27 @@ impl Iroha {
     #[cfg(feature = "telemetry")]
     async fn start_telemetry(
         logger: &LoggerHandle,
-        config: &TelemetryConfiguration,
+        config: &Config,
     ) -> Result<TelemetryStartStatus> {
-        #[allow(unused)]
-        let (config_for_regular, config_for_dev) = config.parse();
-
         #[cfg(feature = "dev-telemetry")]
         {
-            if let Some(config) = config_for_dev {
+            if let Some(config) = &config.dev_telemetry {
                 let receiver = logger
                     .subscribe_on_telemetry(iroha_logger::telemetry::Channel::Future)
                     .await
                     .wrap_err("Failed to subscribe on telemetry")?;
-                let _handle = iroha_telemetry::dev::start(config, receiver)
+                let _handle = iroha_telemetry::dev::start(config.file.clone(), receiver)
```

Comment:
what is this `.file`? Weird

Reply:
Actually it's better to hide this detail by just passing `config.clone()`, agree (if understood correctly)